config/initializers/flipper.rb

```diff
@@ -5,4 +5,8 @@
   # aren't running many betas at the time of writing. If Flipper is used
   # extensively in the future, this can be changed
   config.flipper.preload = false
+
+  # Setting to `true` or `:raise` will raise error when a feature doesn't exist.
+  # Use `:warn` to log a warning instead.
+  config.flipper.strict = false
```

Comment:
The default is `!Rails.env.production?` but my understanding is that we are adding feature flags ad-hoc in development so we need this set to `false`. Let me know otherwise.